hypervm/httpdocs/lib/vps/driver/vps__xenlib.php

```diff
@@ -858,7 +858,7 @@ public function copyKernelModules()
 		if (!stripos($pygrub_record[3], 'pygrub') !== FALSE) {
 
 			$mountpoint = $this->mount_this_guy();
-			if (is_centosfive()) {
+			if (is_centosfive() || is_centosseven()) {
```

Comment:
Hmm, this is so legacy, I think we shouldn't be worried and should not copy kernel modules and kernel image back to domU. We should stick to the pygrub only and get rid of the start Linux domU s with a kernel that lies in the filesystem of the dom0

Reply:
Well i agree but for now im trying to make it work write proper configuration and find out step by step what needs to be fixed. Afterwards we will discuss what we need to keep.